source/_integrations/plugwise.markdown

```diff
@@ -30,115 +30,103 @@ ha_platforms:
+[Plugwise](https://www.plugwise.com) provides smart home climate and power monitoring devices. This integration allows you to monitor and control your climate, energy consumption (including gas) consumption, and energy production. The energy information can be used for the [energy dashboard](/home-energy-management).
 
-Platforms available - depending on your Smile and setup include:
+## Supported devices
 
- - `climate` (for the stand-alone Anna, for Adam, a climate entity is shown for each zone containing devices like an Anna or another type of wired-thermostat, Jip or Lisa combined with one or more Tom/Floor devices)
- - `binary_sensor` (for showing the status of e.g. domestic hot water heating or secondary heater)
- - `button` (for the Adam and the non-legacy Anna and P1 gateways)
- - `number` (for changing a boiler setpoint, a temperature offset)
- - `sensor` (for all relevant products including the Smile P1)
- - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
- - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)
+This integration supports one or more of the Plugwise Gateways available on your network. You can connect to these gateways using your browser, the Plugwise App, or this Home Assistant integration. There are 5 types of gateways:
 
+- Full zonecontrol using the [Adam](https://www.plugwise.com/en_US/zonecontrol) using [additional devices](#devices-overview) such as smart valves and smart-plugs.
+- A stand-alone smart thermostat called [Anna](https://www.plugwise.com/en_US/products/anna).
+- For power monitoring there is a device simply called the [P1](https://www.plugwise.com/en_US/products/smile-p1).
+- Although no longer sold, there also is support for Stretch, a gateway to create network connectivity for their older power products.
+- **Not supported yet!** The newest in the family, [Anna P1](https://www.plugwise.com/product/anna-p1/) cleverly uses energy (solar) information to transfer abundant energy in heatpump usage. [Contact us](#anna-p1) if you have one!
+
+Plugwise formerly sold power-based products using a USB stick as the controller. This integration does not support the `Stick` directly, see [legacy power devices](#legacy-power-devices) for more information.
+
+## Platforms
+
+Depending on your specific gateway and connected devices, the following platforms will be available:
+
+- [Climate](#climate) for Adam and (a stand-alone) Anna.
+- [Binary Sensor](#binary-sensor) for status of your domestic hot water or secondary heater.
+- [Button](#button) to reboot your Plugwise Gateway.
+- [Number](#number) to change a boiler setpoint or temperature offset.
+- [Sensor](#sensor) a variety of sensors is available for all gateways and connected devices.
+- [Select](#select) to change your thermostat schedule or regulation mode.
+- [Switch](#switch) allowing plugs to be commanded.
```

Comment:
This is not how we want to document these. For more information see: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions